### PR TITLE
Xbeinfo

### PR DESCRIPTION
**Pull request by donwayo**

### CODE REVIEW

File: src/Common/XbePrinter.cpp [hunk 1]
```diff
@@ -378,6 +379,21 @@ std::string XbePrinter::GenAlternateSignatureKeys()
     return text;
 }
 
+std::string XbePrinter::GenExtraInfo()
+{
+    std::stringstream text;
+    SSTREAM_SET_HEX(text);
+    if (Xbe_certificate->dwSize == 0x1EC)
```
Comment (PatrickvL) [inline src/Common/XbePrinter.cpp]: Better check greater or equal

File: src/Common/XbePrinter.cpp [hunk 2]
```diff
@@ -378,6 +379,21 @@ std::string XbePrinter::GenAlternateSignatureKeys()
     return text;
 }
 
+std::string XbePrinter::GenExtraInfo()
+{
+    std::stringstream text;
+    SSTREAM_SET_HEX(text);
```
Comment (PatrickvL) [inline src/Common/XbePrinter.cpp]: If you're applying the other suggestion, you could at the same time move this line inside the if statement block

**Comment by PatrickvL** [comment]
Thanks for your contribution! Hopefully more will follow?!?

**Comment by PatrickvL** [comment]
Thanks!!

**Comment by PatrickvL** [comment]
Oh, does this relate to any pending issue? If so, please mention it here (with a #hash sign, so it becomes a bi-directional link) and evaluate how much would still need to be done.

**Comment by donwayo** [comment]
Hi Patrick! No, it doesn't relate with any pending issue, but having this information in the dump might allow one to verify the dump data with the signature key :)

**Pull request merged**